cypress/e2e/flow-builder/connections/creating-removing.cy.ts

```diff
@@ -0,0 +1,41 @@
+describe('Creating a connection', () => {
+  beforeEach(() => {
+    cy.createFlow({
+      label: 'Creating a connection',
+    })
+
+    cy.addBlock(['Content', 'Message']).then(blockId => {
+      cy.get(`[data-cy="block--${blockId}"]`).as('firstBlock')
+    })
+
+    cy.addBlock(['Content', 'Numeric Response']).then(blockId => {
+      cy.get(`[data-cy="block--${blockId}"]`).as('secondBlock')
+    })
+
+    // wait for flow validation after adding blocks
```

Comment:
what if we don't add this? Could you add that in the comment for more context pls?

Reply:
This was to wait for the flow validation panel to disappear because it was overlapping the new block.
But with the block centering fixed it's not longer necessary.
Removed now.